src/vs/workbench/parts/tasks/electron-browser/task.contribution.ts

```diff
@@ -290,7 +290,7 @@ class TaskStatusBarItem extends Themable implements IStatusbarItem {
 	public render(container: HTMLElement): IDisposable {
 
 		let callOnDispose: IDisposable[] = [];
-		const element = document.createElement('a');
+		const element = document.createElement('span');
```

Comment:
@LeonardoBraga this one needs to be an anchor because it is clickable

Reply:
My apologies - I missed the click handler in the code right below.